.eslintignore

```diff
@@ -0,0 +1 @@
+apps/public-reference/public/auth/oidc-client.rsa256.slim.min.js
```

Comment:
That's a good question - I can look more into whether that's possible.

Some background for why this exists:

The silent sign in method works by opening up a background iFrame and attempting an auth request for a new access token with it (this is a standard methodology to avoid insecurely storing refresh tokens in a public app with OIDC). The redirect page that opens up in that invisible iFrame needs to communicate back to the main frame with the response. I was originally doing this using the same `Redirect` component that I'm using for the regular login flow, which works nicely within the React app. This approach worked totally fine in Firefox and Safari. Unfortunately, in Chrome, for some reason as soon as the iFrame communicates back to the parent, the CPU use of the tab spikes to 100% and everything freezes (no DOM interactions possible at all, the debugger doesn't have the DOM loaded - I need to use the task manager to kill the tab). After much trial and error, I was able to get the silent sign in working in chrome by creating a redirect page that doesn't involve React, hence this `public/auth` directory. The problem is that since it needs to run outside of the React context, it doesn't have access to the build pipeline of dependencies.

I don't *really* know why loading the app in the iFrame was messing things up in Chrome, but my gut instinct is that it's probably something to do with how next.js pre-rendering is resolving things </hand-waving>. I did find some reference to a similar problem in [this issue](https://github.com/IdentityModel/oidc-client-js/issues/462#issuecomment-355963315) describing an Angular implementation, and one of the core maintainers mentioned that a simple HTML file for the redirect was the "preferred solution" anyway.

One idea would be to add a CDN dependency for the OIDC client library... but that also runs into potential versioning mis-match issues.

It's possible that this callback file could access the main build file exported by next.js, although I'm not really sure how to do that... and it would also require `Oidc` to be added as a global to `window`, which isn't happening with ES6-style imports in the React app right now.

**Basically, I don't love this solution, but it seemed like the easiest of the sub-optimal options to me. Open to suggestions from the team!**